slider.js

```diff
@@ -0,0 +1,159 @@
+const DEFAULT_CONFIG = {
+  showNavigationButtons: false,
+  autoScroll: true,
+  placeholder: "https://via.placeholder.com/400x200",
+  intervalTime: 3000,
+};
+const SLIDER_CLASSES = {
+  SLIDER_CONTAINER: ".slider-container",
+  LIST: ".slider-list",
+  ITEM: ".slider-item",
+  ITEM_ACTIVE: "slider-item-active",
+  PLACEHOLDER: ".placeholder",
+  SHOW_PLACEHOLDER: "show-placeholder",
+  NAVIGATION_BUTTONS: ".slider-navigation",
+  BTN_PREV: ".slider-navigation-button-prev",
+  BTN_NEXT: ".slider-navigation-button-next",
+};
+/**
+ * Class responsible for handling the slider initialization
+ */
+class Slider {
```

Comment:
1. Could you please create destroy method that will be clearing timer and event listeners?
2. Have you heard something about access modificators (public, private and protected)? Originally JS doesn't support it well, but it would be nice if you understand them, because JS out of the box makes all methods public and it doesn't allow to hide implementation (encapsulation principle of OOP). Also there is a # operator. Just read about it, no need to add it. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Using_Classes#private_fields